fearless/Common/Extension/UIKit/TriangularedButton+Style.swift

```diff
@@ -2,6 +2,16 @@ import Foundation
 import SoraUI
 
 extension TriangularedButton {
+    override var isEnabled: Bool {
+        didSet {
```

Comment:
I think we can have problems with this. Swift docs also doesn't like this: Extensions can add new computed properties, but they can’t add stored properties, or add property observers to existing properties. https://docs.swift.org/swift-book/LanguageGuide/Extensions.html#//apple_ref/doc/uid/TP40014097-CH24-ID151

Reply:
Let's try another way